grafana-plugin/src/components/ManualAlertGroup/ManualAlertGroup.tsx

```diff
@@ -122,41 +122,32 @@ const ManualAlertGroup: FC<ManualAlertGroupProps> = (props) => {
             <LoadingPlaceholder text="Loading..." />
           ) : selectedTeamDirectPaging ? (
             <VerticalGroup>
-              <Label>Team will be notified according to the integration settings:</Label>
+              <Label>Integration to be used for notification</Label>
               <ul className={cx('responders-list')}>
                 <li>
                   <HorizontalGroup justify="space-between">
                     <HorizontalGroup>
-                      {escalationChainsExist && (
```

Comment:
why is this removed?

Reply:
This is related to https://github.com/grafana/oncall/pull/2537#discussion_r1264221326, the PR udpates which integrations are considered "bad" or non-configured, so now this tooltip doesn't convey any additional information, so I removed it.